src/server/patient/patient.config.js

```diff
@@ -20,7 +31,7 @@ let routes = [
 			name: 'given',
 			type: 'string'
 		}],
-		scopes: [],
+		scopes: scopes,
```

Comment:
Is the `/dstu2/patient` search endpoint useable at all by anyone with the `patient/*.*` set of scopes who can only see their own patient record?  What is the expected behavior in the spec if a patient accesses the search endpoint? Should it throw a 403 error, or return the patient record if it matches and no records otherwise?